.github/workflows/icfy-stats.yml

```
@@ -0,0 +1,46 @@
+name: Calculate ICFY stats
+
+on: [push]
+   
+jobs:
+  build:
+    name: Build ICFY stats
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Set up Node
+      uses: actions/setup-node@v1
+      with:
+        node-version: '>=12.15.0'
+    - name: Checkout code
+      uses: actions/checkout@master
```

Maybe this? We need to fetch the refs, not just the commits:
```suggestion
      uses: actions/checkout@v2
      run: git fetch --no-tags --prune --depth=1 origin +refs/heads/master:refs/remotes/origin/master
```
Maybe we can drop the with/fetch-depth as well.

https://github.com/actions/checkout#fetch-all-tags

---

trying this one https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches